core/scripts/sql/texera_ddl.sql

```diff
@@ -154,6 +154,7 @@ CREATE TABLE IF NOT EXISTS workflow_executions
     `eid`             INT UNSIGNED AUTO_INCREMENT NOT NULL,
     `wid`             INT UNSIGNED NOT NULL,
     `vid`             INT UNSIGNED NOT NULL,
+    `uid`             INT UNSIGNED NOT NULL,
```

Comment:
I believe this will cause a problem in `development mode` when the option of uID is null and not available. Maybe allow NULL.

Reply:
Also, you need to add a reference of foreign key to USER table